changes/5434.fixed

```diff
@@ -0,0 +1 @@
+Fixed health check for beat container in `docker-compose.yml` under `docker-compose` v1.x.
```

Comment:
rename to `5434.housekeeping`?

Reply:
Possibly... really the change to add the healthcheck for #5428 should have been under `housekeeping` as well.  Let's merge this and I'll fix it up in the release branch.